application/features/mfa.feature

```diff
@@ -60,8 +70,20 @@ Feature: MFA
     When I request "/mfa" be created
     Then the response status code should be 400
 
-  Scenario: Create new MFA record of type webauthn
-#TODO - create tests that make use of the webauthn client
+  Scenario: Request a MFA record of type webauthn
+    When the user requests a new webauthn MFA
+    Then the response status code should be 200
+    And the response body should contain 'publicKey'
+    And the response body should contain 'challenge'
+
+  Scenario: Verify a new MFA webauthn registration
+    Given the user has requested a new webauthn MFA
```

Comment:
Let's revise this to be clearly intended for testing invalid values, perhaps like this:
```suggestion
  Scenario: Attempt to verify a new but invalid MFA webauthn registration
    Given the user has requested a new webauthn MFA with all the required fields but invalid values
```